style.css

```diff
@@ -0,0 +1,1878 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700&display=swap");
+
+*,
+::before,
+::after {
+  box-sizing: border-box;
+
+  /* 1 */
+  border-width: 0;
+
+  /* 2 */
+  border-style: solid;
+
+  /* 2 */
+  border-color: #e5e7eb;
+
+  /* 2 */
+  --tw-border-spacing-x: 0;
+  --tw-border-spacing-y: 0;
+  --tw-translate-x: 0;
+  --tw-translate-y: 0;
+  --tw-rotate: 0;
+  --tw-skew-x: 0;
+  --tw-skew-y: 0;
+  --tw-scale-x: 1;
+  --tw-scale-y: 1;
+  --tw-scroll-snap-strictness: proximity;
+  --tw-ring-offset-width: 0;
+  --tw-ring-offset-color: #fff;
+  --tw-ring-color: rgb(59 130 246 / 50%);
+  --tw-ring-offset-shadow: 0 0 #0000;
+  --tw-ring-shadow: 0 0 #0000;
+  --tw-shadow: 0 0 #0000;
+  --tw-shadow-colored: 0 0 #0000;
+}
+
+::before,
+::after {
+  --tw-content: "";
+}
+
+html {
+  line-height: 1.5;
+  tab-size: 4;
+
+  /* 3 */
+  font-family:
+    lato,
+    system-ui,
+    -apple-system,
+    BlinkMacSystemFont,
+    "Segoe UI",
+    Roboto,
+    "Helvetica Neue",
+    Arial,
+    "Noto Sans",
+    sans-serif,
+    "Apple Color Emoji",
+    "Segoe UI Emoji",
+    "Segoe UI Symbol",
+    "Noto Color Emoji";
+
+  /* 4 */
+  font-feature-settings: normal;
+
+  /* 5 */
+}
+
+/*
+1. Remove the margin in all browsers.
+2. Inherit line-height from `html` so users can set them as a class directly on the `html` element.
+*/
+
+body {
+  margin: 0;
+
+  /* 1 */
+  line-height: inherit;
+
+  /* 2 */
+  position: relative;
+}
+
+.header-top-inner,
+.navbar-inner,
+.container,
+.aboutcontainer {
+  width: 83.5%;
+}
+
+.container {
+  background: url("./public/images/header-bg.jpg");
+  margin-top: 22px;
+  z-index: -99;
+}
```

Comment:
- [ ] Kindly make sure the background image of your container covers the whole space on the screen inorder to meed the design. Please make it covers even the white space on both ends of the container. 

    - **YOUR DESIGN**

        ![image](https://user-images.githubusercontent.com/87197412/210709563-d0680ea7-5d4e-4ed1-ba54-c29015e203dd.png)

      ![image](https://user-images.githubusercontent.com/87197412/210709618-cee273cc-01c4-49fa-8b84-a8c23086671e.png)


   - **PROVIDED DESIGN**
   
     ![image](https://user-images.githubusercontent.com/87197412/210709723-30021662-d429-4bae-bb1b-6e2430f2e9ac.png)

      ![image](https://user-images.githubusercontent.com/87197412/210709784-d98a897c-479b-46bf-971b-4d844467fe39.png)